pkg/controllers/uiplugin/distributed_tracing.go

```diff
@@ -29,13 +30,36 @@ func createDistributedTracingPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace,
 		extraArgs = append(extraArgs, fmt.Sprintf("-features=%s", strings.Join(features, ",")))
 	}
 
+	proxyName, proxyNamespace := "distributed-tracing-console-plugin", "openshift-tracing"
```

Comment:
```suggestion
	defaultServiceName, defaultServiceNamespace := "distributed-tracing-console-plugin", "openshift-tracing"
```

Reply:
Name and namespace are passed in as parameters to the `createDistributedTracingPluginInfo` function and we should use them so that the service and the proxy don't get out of sync.

Reply:
I updated this in the latest commit.